frontend/src/index.html

```diff
@@ -200,6 +202,11 @@ <h3 class="flex-grow break-words font-semibold" title="{{title}}">{{#helpers.hig
                     </a>
                   </div>
                 </div>
+                <div x-show="is_open" @click="text_open=false" class="mt-1">
+                  {{#_highlightResult.topics}}
+                  <h3 title="{{topics}}">{{#helpers.highlight}} { "attribute": "title" }{{/helpers.highlight}}>{{{value}}}</h3>
```

Comment:
- It shouldn't be `h3`. Use `p` instead.
- Topics are too large to use it in `title` attribute. Better to put some explanatory text like “Topics discussed in the recording” there.
- `{{#helpers.highlight}} { "attribute": "title" }{{/helpers.highlight}}` seem to be excessive. As soon as you have already opened `{{#_highlightResult.topics}}` section, `{{{value}}}` should be enough.
- There is a `>` symbol which seems to be a typo. Moreover, HTML linter warns about it: 
![image](https://user-images.githubusercontent.com/6065071/115958427-ada22f80-a531-11eb-9584-b676df0e02cc.png)